components/Press/PressLinks/PressLinks.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './PressLinks.css';
+import * as Articles from './Articles';
+import ArticleGroup from './ArticleGroup/ArticleGroup';
+
+PressLinks.propTypes = {
+  numberOfInitiallyVisibleLinks: PropTypes.number,
+};
+
+PressLinks.defaultProps = {
+  numberOfInitiallyVisibleLinks: 5,
+};
+
+function PressLinks({ numberOfInitiallyVisibleLinks }) {
+  return (
+    <div className={styles.logos}>
+      <div className={styles.flexContainer}>
+        {Object.keys(Articles).map(group => (
+          <ArticleGroup
+            key={`ArticleGroup_${group}`}
+            title={group}
+            links={Articles[group]}
```

Comment:
ohhhhh I finally see..

What if you just called `title` --> `region` + `links` --> `articles` instead

Reply:
that does make a bit more sense.

Reply:
tbh naming stuff has never been my strong area, hehe